src/app/components/common/alert-dialog/alert-dialog.component.ts

```diff
@@ -23,6 +23,24 @@ export class AlertDialogComponent implements OnInit {
 
   ngOnInit() { }
 
+  ngOnChanges() {
+    if (this.childData.key === 'restoreBackup') {
+      this.childData.action = 'Restore';
+    }
+    if (this.childData.key === 'deleteBackup' || this.childData.key === 'deleteCertificate' || this.childData.key === 'deleteUser') {
+      this.childData.action = 'Delete';
+    }
+    if (this.childData.key === 'logout' || this.childData.key === 'clearSessions') {
+      this.childData.action = 'Logout';
+    }
+    if (this.childData.key === 'clearSessions') {
+      this.childData.action = 'Clear Sessions';
+    }
+    if (this.childData.key === 'createBackup') {
+      this.childData.action = 'Create';
```

Comment:
ACTION dispatcher should be in SCREAMING_SNAKE_CASE e.g. CLEAR_SESSION

Reply:
This action is to be displayed as a `text on button` in alert dialog; just like:

![alert-dialog](https://user-images.githubusercontent.com/29797330/45203228-13feb480-b299-11e8-82af-8dc6ab9843f4.png)

So can not be in SNAKE_CASE

Reply:
you mean this not action itself, but actionButtonValue ?


BTW, in this screenshot there is one extra space before ` user`, it should not be